packages/cli-tool/package.json

```diff
@@ -1,18 +1,21 @@
 {
-  "name": "@nimblehq/react-templates",
-  "version": "0.1.0",
+  "name": "@nimblehq/react-template",
+  "version": "1.0.0",
   "description": "Nimble React Application Generator",
   "author": "Nimble <dev@nimblehq.co>",
   "bin": {
-    "nimble-react ": "./bin/run"
+    "nimble-react": "./bin/run"
   },
   "keywords": [
     "oclif"
   ],
   "homepage": "https://github.com/nimblehq/react-templates/packages/cli-tool",
   "license": "MIT",
   "main": "dist/index.js",
-  "repository": "nimblehq/react-templates",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nimblehq/react-templates.git"
```

Comment:
Can we not do `https://github.com/nimblehq/react-templates.git` directly here?

Reply:
Agreed, revert in [84fb583](https://github.com/nimblehq/react-templates/pull/114/commits/84fb58317cb38c5ad732678fba5d197fc3dd2d48)